libs/sfm/bundler_init_pair.cc

```diff
@@ -84,6 +84,26 @@ InitialPair::compute_pair (Result* result)
         if (angle < this->opts.min_triangulation_angle)
             continue;
 
+        /* Run triangulation to ensure correct pair */
+        Triangulate::Options triangulate_opts;
+        Triangulate::Statistics stats;
+        Triangulate triangulator(triangulate_opts);
+        std::vector<CameraPose const*> poses;
+        poses.push_back(&pose1);
+        poses.push_back(&pose2);
+        std::size_t successfull_triangulations = 0;
+        for (std::size_t j = 0; j < candidate.matches.size(); ++j)
+        {
+            std::vector<math::Vec2f> positions;
```

Comment:
I think it's more efficient if you keep this outside the loop and resize(2), then just assign in the loop.